register.js

```diff
@@ -0,0 +1,26 @@
+const register = require("@swc/register/lib/node").default;
+
+register({
+  jsc: {
+    parser: {
+      syntax: "typescript",
+      dynamicImport: false,
+      decorators: false,
+      hidden: {
+        jest: true,
+      },
+    },
+    transform: null,
+    target: "es2021",
```

Comment:
How did we choose this? Any reason not to jump straight to 2022?

Reply:
I cargo-culted it from the `test-app` directory like the grown up developer I am 🤣 

I'll update it to es2022